vms/evm/database/wrapped_database.go

```diff
@@ -0,0 +1,61 @@
+// Copyright (C) 2019-2025, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package database
+
+import (
+	"errors"
+
+	"github.com/ava-labs/libevm/ethdb"
+
+	"github.com/ava-labs/avalanchego/database"
+)
+
+var (
+	_ ethdb.Batch         = (*ethBatchWrapper)(nil)
+	_ ethdb.KeyValueStore = (*ethDBWrapper)(nil)
+
+	ErrSnapshotNotSupported = errors.New("snapshot is not supported")
+	ErrStatNotSupported     = errors.New("stat is not supported")
```

Comment:
It seems like we don't care about these error types - I would just unexport here to remove some package api bloat.